ci/before_deploy

```diff
@@ -17,8 +17,10 @@ fi
 # Generate random key & jwt for Mercure if not set
 if [[ -z $MERCURE_JWT_KEY ]]; then
     npm install --global "@clarketm/jwt-cli"
-    export MERCURE_JWT_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-    export MERCURE_JWT=$(jwt sign --noCopy '{"mercure": {"publish": ["*"]}}' $MERCURE_JWT_KEY)
+    MERCURE_JWT_KEY=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+    export MERCURE_JWT_KEY
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
We should add shellcheck to this repo btw